README.md

```diff
@@ -870,6 +871,7 @@ Table of Contents
 
 ## Visitor Session Recording
 
+   * [howuku.com](https://howuku.com) — 200 sessions/month with 1 month data retention.
```

Comment:
I appreciate your tool does a lot and do belong to both sections, but we tend to list only major things like github in multiple sections, so please pick one - else the list gets emmense

Reply:
Thanks for responding to my request. As per request, I've remove howuku listing from Visitor Session Recording section.